packages/concerto-cto/package.json

```diff
@@ -38,24 +38,21 @@
   "author": "accordproject.org",
   "license": "Apache-2.0",
   "devDependencies": {
-    "at-least-node": "1.0.0",
     "chai": "4.3.6",
     "chai-as-promised": "7.1.1",
     "chai-things": "0.2.0",
-    "debug": "4.3.4",
-    "eslint": "8.2.0",
-    "jsdoc": "^4.0.2",
-    "mocha": "10.0.0",
-    "nyc": "15.1.0",
-    "peggy": "2.0.1",
-    "sinon": "12.0.0",
-    "sinon-chai": "3.7.0",
-    "typescript": "5.6.3"
+    "debug": "4.3.7",
+    "eslint": "8.57.1",
+    "jsdoc": "^4.0.4",
+    "mocha": "10.8.2",
+    "nyc": "17.1.0",
+    "peggy": "4.2.0",
```

Comment:
@dselman @mttrbrts do we want to update the version for peggy? It's leading to quite a lot of changes to the generated parser file. Although all the unit tests are passing.

Reply:
I've reviewed the release notes for v3.0 and v4.0 of peggy, and am comfortable that the changes don't affect us.

Providing that the tests and build pass, I'm happy to proceed.